app/src/androidTest/java/com/nervousfish/nervousfish/test/first_time_use.feature

```diff
@@ -0,0 +1,7 @@
+Feature: First time use
```

Comment:
Oops, that was because I had to rerun that cucumber test quite a few times and I didn't want to wait for those other tests